bolt/test/X86/internal-call-instrument-so.s

```diff
@@ -5,7 +5,7 @@
 # RUN: llvm-mc -filetype=obj -triple x86_64-unknown-unknown %s -o %t.o
 # Delete our BB symbols so BOLT doesn't mark them as entry points
 # RUN: llvm-strip --strip-unneeded %t.o
-# RUN: ld.lld %t.o -o %t.exe -q -shared -fini=_fini
+# RUN: ld.lld %t.o -o %t.exe -q -shared -fini=_fini -init=_init
```

Comment:
Sorry for jumping in late. Is the change to this test supposed to check the new functionality? If so, perhaps worth adding `FileCheck`?